pontoon/base/static/js/l20n_interface.js

```diff
@@ -0,0 +1,313 @@
+/* Public functions used across different files */
+var Pontoon = (function (my) {
+  return $.extend(true, my, {
+    l20n: {
+
+      /*
+       * Populate FTL translation area with existing data
+       */
+      renderEditor: function (translation) {
```

Comment:
I agree. Let's take care of these kinds of ugliness as part of React rewrite effort. Stas already played with it.